client/src/components/Typography/Heading.js

```diff
@@ -14,22 +14,22 @@ const Heading = styled(Title)`
   }
 
   &.h1 {
-    font-size: 4.2rem;
+    font-size: ${theme.typography.heading.one};
```

Comment:
Let's destructuring `heading` like this:

`const { heading } = theme.typography`

and then replace wherever it's use, for example like this `heading.one`

Reply:
ah thanks! i was wondering how we can shorten it